sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -142,6 +166,10 @@ private Crashes() {
         mCrashesListener = DEFAULT_ERROR_REPORTING_LISTENER;
         mUnprocessedErrorReports = new LinkedHashMap<>();
         mErrorReportCache = new LinkedHashMap<>();
+        HandlerThread thread = new HandlerThread(THREAD_NAME);
```

Comment:
We are free from thread pool limitation with Handler. It is an SDK and trying not to consume resources as many as possible. Going with AsyncTask is not a problem since it is one shot tasks as you mentioned.